proto/process/agent.proto

```diff
@@ -720,6 +720,21 @@ message Connection {
 	uint32 stateIndex = 46;
 }
 
+message FailedConnection {
+	int32 pid = 1;
+	Addr laddr = 2; // Local address
+	Addr raddr = 3; // Remote address
+
+	ConnectionFamily family = 4;
+	ConnectionType type = 5;
+	ConnectionDirection direction = 6;
+	uint32 netNS = 7; // Network namespace
+	EphemeralPortState isLocalPortEphemeral = 8;
+
+	uint64 failureCount = 9;
+	int32 lastErrno = 10;
```

Comment:
I'm not sure what this is from the proto file alone, it could use a comment

Reply:
errno may not be very portable outside of linux.

Reply:
Changed it to a protobuf enum.